### PR TITLE
Add new readme 4 with 8 lines

### PR DESCRIPTION
**Pull request by jkwongit**
Create new PR for review

### CODE REVIEW

File: readme_4.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+add line 1
+add line 2
+add line 3
+add line 4
+add line 5
```
Comment (hkjackjoke) [inline readme_4.txt:5]: Line 2 to 5 need change.

Comment (hkjackjoke) [reply readme_4.txt:5]: Go to change line 2 ~ 5.

Comment (jkwongit) [reply readme_4.txt:5]: I don't think any problem.

Comment (hkjackjoke) [reply readme_4.txt:5]: Ha Ha, see below:
https://github.com/hkjackjoke/HelloWorld/blob/5ec1d2a4978f6192c426f23145a22aa079372a9b/readme_4.txt#L1-L3

Comment (jkwongit) [reply readme_4.txt:5]: Eat this
image

Comment (hkjackjoke) [reply readme_4.txt:5]:
```suggestion
add line 2
add line 3
add line 4
add line 5

```
add line 2b
add line 4c

File: readme_4.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+add line 1
+add line 2b
+add line 3
+add line 4
```
Comment (hkjackjoke) [inline readme_4.txt:4]:
```suggestion
add line 88
add line 3
add line 328

```

File: readme_4.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+add line 1
+add line 88
+add line 169
+add line 4
+add line 5
```
Comment (hkjackjoke) [inline readme_4.txt:5]:
```suggestion
add line 2b
add line 169
add line 4
add line 555

```

Comment (jkwongit) [reply readme_4.txt:5]: ### aaa
**bbb**
_ccc_

> ddd
`eee`
[ccs](https://web.dev/learn/css/the-cascade/)

- 111
- 222

1. 333
2. 444

- [ 1] T1
- [ ] T2

- [T3 ] 
#19

Comment (hkjackjoke) [reply readme_4.txt:5]: Duplicate of #12

| Header | Header | Header |
|--------|--------|--------|
| Cell | Cell | Cell |
| Cell | Cell | Cell |

Comment (jkwongit) [reply readme_4.txt:5]: ~~ ~~Enough~~ ~~

Comment (hkjackjoke) [reply readme_4.txt:5]: lucky cat

**Pull request closed without merging**